twisted-infra/twisted-buildbot-configuration
# Move all sources to git

# Problem

Current sources are using a mixture of git and bzr . We should move all them to git to reduce the number of VCS required on a slave
# Changes

Moved our bzr to git and clean bzr/svn code.

As I drive by I have cleaned the master.cfg based on pyflakes output
# How to test

Please see https://github.com/twisted-infra/braid/issues/72

This needs the production slaves to be updated with git... and then apply the changes and buildmaster and restart the buildmaster.

```
fab buildbot.update
fab buildbot.restart
```
